docs/BUILD

```diff
@@ -246,11 +246,15 @@ pkg_tar(
 genrule(
     name = "html_release",
     outs = ["html_release.tar.gz"],
+    # BUILD_SHA must be set in release builds
```

Comment:
Will/can the script fail if it is not set?

Reply:
yeah it will - bazel enforces set env vars so it fails

Reply:
(ditto re _too many vars_)